src/stubs/navBarData.js

```diff
@@ -50,10 +57,15 @@ export const navBarData = [
 ];
 
 export const navBarCartData = {
-  cartLabel: 'MY Files',
+  // cartLabel: 'My Files',
```

Comment:
Looks Perfect to me. 

@AjayDoddapaneni Do you think should we remove commented code from here ?